core/git.js

```diff
@@ -40,3 +40,7 @@ exports.getCurrentBranchName = function(callback) {
 exports.clone = function(url, dir, callback) {
     command(['clone', url, dir], callback);
 };
+
+exports.submoduleUpdate = function(callback) {
+    command(['submodule', 'update', '--init', '--recursive'], callback);
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 2 space characters but found 4.](https://www.codacy.com/app/theprojectshop/Kassy/file/2554993051/issues/source?bid=3282987&fileBranchId=3284644#l45)